ios/reactNativeApp.xcodeproj/project.pbxproj

```diff
@@ -468,6 +472,7 @@
 				898C440300784E69B91D0ED6 /* Roboto-Medium.ttf in Resources */,
 				21C5EFC22189F86200F3C098 /* package.json in Resources */,
 				FA705002A8FD4F5BBDB4B4F4 /* Roboto-Regular.ttf in Resources */,
```

Comment:
Should that be somehow removed?

Reply:
Actually, why is Roboto even linked for iOS? It's not used, right?

Reply:
I think at some historical point we probably linked it, and autolinking does not remove it. 

This does not `bubble` up to the kiwi.com app anyways.

Reply:
But sure, we could add an issue to remove it.